.goreleaser.yml

```diff
@@ -134,25 +110,6 @@ archives:
       - LICENSE
       - completions/*
 
-brews:
-  - name: regen-ledger
-    ids: 
-      - regen
-    tap:
-      owner: regen-network
-      name: regen-ledger
-      branch: homebrew-taps
-    folder: "{{ .Tag }}"
-    homepage: "https://www.regen.network/"
-    description: "Blockchain-powered marketplace for climate solutions and planetary regenerations"
-    test: |
-      system "#{bin}/regen --version"
-    install: |
-      bin.install "regen"
-      bash_completion.install "completions/regen.bash" => "regen"
-      zsh_completion.install "completions/regen.zsh" => "_regen"
-      fish_completion.install "completions/regen.fish"  
```

Comment:
Same here. Would be nice to keep a working setup in the file prior to the issue.